packages/mesos/extra/patches/0001-Set-LIBPROCESS_IP-into-docker-container.patch

```diff
@@ -0,0 +1,28 @@
+From 0f762c15455900083f7a3f72c6cdc60a418ec12f Mon Sep 17 00:00:00 2001
```

Comment:
Does it make sense / can we remove this line? It seems to me that there is no value in this info.

Reply:
It is what git-format reported. I am not sure what's the convention there.

Reply:
Could we please just use whatever `git-format` reports? This e.g., allows to construct proper history with `git-am` (which would be easier to correlate with upstream).

`patch` will be fine with a surprisingly large collection of formats.

Reply:
I'd prefer to stick with `git-format` output -- fewer troubles that way.